config/src/config/consensus_observer_config.rs

```diff
@@ -42,11 +47,13 @@ impl Default for ConsensusObserverConfig {
             observer_enabled: false,
             publisher_enabled: false,
             max_network_channel_size: 1000,
-            max_subscription_timeout_ms: 30_000,   // 30 seconds
-            max_synced_version_timeout_ms: 60_000, // 60 seconds
-            peer_optimality_check_interval_ms: 300_000, // 5 minutes
-            progress_check_interval_ms: 5_000,     // 5 seconds
-            request_timeout_ms: 10_000,            // 10 seconds
+            max_parallel_serialization_tasks: num_cpus::get(), // Default to the number of CPUs
```

Comment:
with Move execution looking to target 100+ cores of parallelism, we should start thinking about smarter default configs for this kind of thing?